cortex/content.py

```diff
@@ -53,13 +53,13 @@ def upload(self, key: str, project: str, stream_name: str, stream: object, conte
         .. seealso: uploadStreaming()
         """
         r = tenacity.Retrying(
-            stop = tenacity.stop_after_attempt(retries + 1),
-            retry = tenacity.retry_if_exception(ManagedContentClient._http_request_retry_predicate)
-            )
-        return r.wraps(self._upload)(key, project, stream_name, stream, content_type)
+            stop=tenacity.stop_after_attempt(retries + 1),
+            retry=tenacity.retry_if_exception(ManagedContentClient._http_request_retry_predicate)
+        )
+        return r.wraps(self._upload)(key, self._project, stream_name, stream, content_type)
 
-    def _upload(self, key: str, project: str, stream_name: str, stream: object, content_type: str):
-        uri = self.URIs['content'].format(projectId=project)
+    def _upload(self, key: str, stream_name: str, stream: object, content_type: str):
+        uri = self.URIs['content'].format(projectId=self._project)
```

Comment:
we always seem to access values in `self.URIs` without a `.get('xx', 'default_value')`. Is this because these keys are always present in `self.URIs` ?

Reply:
nvm I see its a constant